search/base.py

```diff
@@ -0,0 +1,18 @@
+"""
```

Comment:
The `_test.py` files are supposed to be located next to the files they test so that they show up next to each other in code review. I'm not sure it makes sense to move `search/base.py` into a `tests` directory by itself